lib/rules/selector-max-pseudo-class/__tests__/index.js

```diff
@@ -0,0 +1,186 @@
+"use strict";
+
+const messages = require("..").messages;
+const ruleName = require("..").ruleName;
+const rules = require("../../../rules");
+
+const rule = rules[ruleName];
+
+testRule(rule, {
+  ruleName,
+  config: [0],
+  skipBasicChecks: true,
+
+  accept: [
+    {
+      code: "",
+      description: "empty stylesheet"
+    },
+    {
+      code: "a {}",
+      description: "empty rule"
+    },
+    {
+      code: '@import "foo.css";',
+      description: "blockless statement"
+    },
+    {
+      code: "a {}",
+      description: "selector with no pseudo class"
+    },
+    {
+      code: ".foo { .bar { .baz {} } }",
+      description: "nested selectors with no pseudo classes"
+    }
```

Comment:
Can we get two simple pseudo-element tests here please, for lines 43-50 of the rule code? e.g.

``` 
{
  code: "a:before {}",
  description: "single level 2 puesdo-element"
},
{
  code: "a::before {}",
  description: "single level 3 puesdo-element"
}
```